apps/web/src/pages/studio-onboarding/preview.tsx

```diff
@@ -0,0 +1,189 @@
+import { Prism } from '@mantine/prism';
+import { Tabs } from '@novu/design-system';
+import { IconCode, IconVisibility } from '@novu/novui/icons';
+import { css } from '@novu/novui/css';
+import { vstack } from '@novu/novui/patterns';
+import { api, ROUTES, useAuth } from '@novu/shared-web';
+import { useEffect, useMemo, useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+import { PreviewWeb } from '../../components/workflow/preview/email/PreviewWeb';
+import { useActiveIntegrations } from '../../hooks/index';
+import { useTemplates } from '../../hooks/useTemplates';
+import { Background } from './components/Background';
+import { Footer } from './components/Footer';
+import { Header } from './components/Header';
+import { testTrigger } from '../../api/notification-templates';
+import { useMutation } from '@tanstack/react-query';
+import * as Sentry from '@sentry/react';
+import { IStepVariant } from '@novu/shared';
+
+export const StudioOnboardingPreview = () => {
+  const { currentUser } = useAuth();
+  const [tab, setTab] = useState('Preview');
+  const [content, setContent] = useState('');
+  const [subject, setSubject] = useState('');
+
```

Comment:
<!-- Chores are simple tasks that must be done before the subject can be “officially” accepted. Usually, these comments reference some common process. Try to leave a link to the process description so that the reader knows how to resolve the chore.  -->
☑ **chore:** Please use explicit types (`<string>`) for state‏